orchestrator/careplanservice/service.go

```diff
@@ -443,3 +445,51 @@ func (s Service) baseUrl() *url.URL {
 func getResourceType(resourcePath string) string {
 	return strings.Split(resourcePath, "/")[0]
 }
+
+func (s *Service) ensureSearchParameterExists() {
+	searchParamID := "CarePlan-subject-identifier"
+
+	// Define the SearchParameter
+	searchParam := fhir.SearchParameter{
+		Id:          &searchParamID,
+		Url:         "http://zorgbijjou.nl/SearchParameter/CarePlan-subject-identifier",
```

Comment:
I feel like we are limiting ourselves by hardcoding these endpoints and email addresses for future testing, environments, and scaling. Should this not be configurable?

Reply:
We could, but in FHIR it's "just" a [canonical identifier](https://www.hl7.org/fhir/r4/references.html#canonical), so they may/should not change over time. It's also just created once, when the server starts up and it's not present yet.

Reply:
Ok, sounds good to me